app/models/knight.rb

```diff
@@ -1,3 +1,17 @@
 class Knight < Piece
 
+	def is_valid?(new_x, new_y)
+		total_x = new_x - self.x_coordinate
+		total_x = total_x.abs
+		total_y = new_y - self.y_coordinate
+		total_y = total_y.abs
+		if total_x == 1 && total_y == 3

```

Comment:
One thing here:

The whole if/else could be just:
`total_x == 1 && total_y == 2  || total_x == 2 && total_y == 1` 

it will automatically return true or false and replaces 7 lines 😄